yarn.lock

```diff
@@ -12566,6 +12572,37 @@ __metadata:
   languageName: node
   linkType: hard
 
+"mocha@npm:^11.1.0":
```

Comment:
Nit, can we keep the same version of Mocha we're using elsewhere so we can update all at one?

Reply:
Done in 03f321bc2b520ec9af5348727a598f215a4613f9